torch/backends/_nnapi/serializer.py

```diff
@@ -1131,6 +1133,27 @@ def add_qadd(self, node, opcode, fuse_code):
 
         self._do_add_binary(node, opcode, fuse_code, qparams=(scale, zero_point))
 
+    def add_softmax(self, node):
+        assert node.inputsSize() == 3
+        in_id, in_oper = self.get_tensor_operand_by_jitval(node.inputsAt(0))
+
+        _, softmax_dim = self.get_constant_value(node.inputsAt(1), "IntType")
+
+        out_id = self.add_tensor_operand(node.outputsAt(0), in_oper)
+        for dim, size in enumerate(in_oper.shape):
+            if size == 0:
+                self.forward_operand_shape(out_id, dim, in_id, dim)
```

Comment:
Do you think this is a common enough pattern to justify making it a helper function?

Reply:
I think so, I'll add a clean task